toolkits/global/packages/global-forms/demo/context.json

```diff
@@ -84,7 +83,7 @@
 						]
 					},
 					{
-						"template": "select",
+						"template": "globalFormSelect",
 						"label": "Animal (with hint)",
```

Comment:
I think, at some point, we may want to use 'real world' SN content for this. I recall @amyhupe discussing it in a wider Elements ideal.

Reply:
… I think

Reply:
👍🏼